docs/05-technical-reference/00-architecture/assets/evnt-architecture.svg

```diff
@@ -1,3 +1,4 @@
t x="910" y="109" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">6</text></switch></g><rect x="370" y="280" width="64.53" height="384.58" rx="9.03" ry="9.03" fill="#f0f6ff" stroke="none" transform="rotate(-90,402.26,472.29)" pointer-events="all"/><rect x="220" y="445.79" width="113" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 111px; height: 1px; padding-top: 456px; margin-left: 221px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;"><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;"><font style="font-size: 15px"><b>Reference key:</b></font></div></div></div></foreignObject><text x="277" y="459" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">Reference key:</text></switch></g><rect x="292.5" y="465.79" width="306" height="33" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 304px; height: 1px; padding-top: 482px; margin-left: 295px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: left;"><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;"><div><span>Dashed connectors represent background processes</span></div></div></div></div></foreignObject><text x="295" y="486" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px">Dashed connectors represent background processes</text></switch></g><path d="M 221 482.79 L 275.63 482.79" fill="none" stroke="#666666" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><path d="M 280.88 482.79 L 273.88 486.29 L 275.63 482.79 L 273.88 479.29 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Text is not SVG - cannot display</text></a></switch></svg>
```

Comment:
Please note that Kyma guidelines ask for rounded corners in the diagrams: https://kyma-project.io/community/guidelines/content-guidelines/09-diagrams/#examples